include/mega/posix/megawaiter.h

```diff
@@ -23,7 +23,7 @@
 #define WAIT_CLASS PosixWaiter
 
 #include "mega/waiter.h"
-
+#include <mutex>
```

Comment:
This is already included on `develop`. Is your fork out of date?

Reply:
Yes, confirmed, this change was added several days ago, so we could close this issue.
.........
Commits on Oct 22, 2019
add missing include

@polmr
polmr committed 8 days ago